examples/bin/spark-shell-plot.mscala

```diff
@@ -0,0 +1,19 @@
+import org.apache.mahout.visualization.mplot2d
+
+val mxRnd = Matrices.symmetricUniformView(5000000, 2, 1234)
+val drmRand = drmParallelize(mxRnd)
+
+val drmSin = drmRand.mapBlock() {case (keys, block) =>  
+  val blockB = block.like()
+  for (i <- 0 until block.nrow) {
+    blockB(i, 0) = block(i, 0) 
+    blockB(i, 1) = Math.sin((block(i, 0) * 8))
+  }
+  keys -> blockB
+}
+
+new mplot2d(drmRand, samplePercent = 10)
```

Comment:
that is just for illistrituve purposes,  the default is also 10% but the user can override it to be 1 to 100 %